test/com/facebook/buck/android/testdata/android_instrumentation_test_integration_test/apps/myapp_test/BUCK

```diff
@@ -0,0 +1,14 @@
+android_instrumentation_apk(
+  name = 'app_test',
+  manifest = 'AndroidManifest.xml',
+  apk = '//apps/myapp:app-exo',
```

Comment:
That definitely sounds like a nice to have.  But I do agree that's kind of unrelated to this pull request.  I'll try to do it if I find some time.